src/app/application.h

```diff
@@ -131,8 +132,16 @@ private slots:
 #endif
 
 private:
+    struct Params
```

Comment:
Is it possible to have a less generic name?

Reply:
> Is it possible to have a less generic name?

What, for example? It's a private name, so I didn't care much. Besides, I really can't think of anything more suitable for storing parameters.